shared/secrets.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  // converts an object from serverless framework ({"a": "b", "c": "d"})
+  // to an array of secrets expected by the API :
+  // [{"key": "a", "value": "b"}, {"key": "c", "value": "d"}]
+  convertObjectToModelSecretsArray(obj) {
+    if (obj === {} || obj === null || obj === undefined) {
+      return [];
+    }
+    return Object.keys(obj)
+      .map(k => ({
+        key: k,
+        value: obj[k],
+      }));
+  },
+
+  // resolves a value from a secret
+  // if this is a raw value, return the value
+  // if this is a reference to a local environment variable, return the value of that env var
+  resolveSecretValue(key, value) {
+    const envVarRe = /^\${([^}]*)}$/;
+    const found = value.match(envVarRe);
+
+    if (!found) {
+      return value;
+    }
+
+    if (process.env[found[1]] === null || process.env[found[1]] === undefined) {
```

Comment:
equivalent to `if (process.env[found[1]] == null )` no ?

Reply:
Even more readable:

```js
if (found[1] in process.env) {
  return process.env[found[1]];
}
```